src/main/java/com/elytradev/concrete/inventory/gui/client/ConcreteContainerGui.java

```diff
@@ -158,11 +151,15 @@ public void setGuiSize(int w, int h) {
 	protected void drawGuiContainerBackgroundLayer(float partialTicks, int mouseX, int mouseY) {
 		GuiDrawing.drawGuiPanel(guiLeft - PADDING, guiTop - PADDING, xSize + ((PADDING - 1) * 2), ySize + ((PADDING - 1) * 2));
 		
-		if (inventorySlots != null && this.container.getRootPanel() != null) {
+		if (this.container.getRootPanel() != null) {
 			this.container.getRootPanel().paintBackground(guiLeft, guiTop);
 		}
-		
-		//TODO: Change this to a label that lives in the rootPanel instead
-		fontRenderer.drawString(container.getLocalizedName(), guiLeft, guiTop, 0xFF404040);
```

Comment:
Note: This is replaceable with a `WLabel` created by `WLabel.ofInventoryDisplayName(...)` that is added to the root panel.